wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,122 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AESGCM_AEAD
+
+Disables AES GCM algorithm support.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN.
+
+#### NO_CHECK_PRIVATE_KEY
+
+RSA - needed to encrypt salt.
+
+#### NO_DH
+
+Disables DH support.
+
+#### NO_ED25519_CLIENT_AUTH
+
+Disables client authEd25519 requires caching enabled for tracking message hash used in EdDSA_Update for signing.
+
+#### NO_ED448_CLIENT_AUTH
+
+Disables client authentification for ED448.
+
+#### NO_FORCE_SCR_SAME_SUITE
+
+Applys a restriction on secure negotation. Don't allocate Suites' object on renegotiation.
+
+#### NO_MULTIBYTE_PRINT
+
+Used for compiling out special characters that embedded devices may have problems with.
+
+#### NO_OLD_SSL_NAMES
+
+Removes unneeded namespace.
+
+#### NO_OLD_WC_NAMES
+
+Removes unneeded namespace.
+
+#### NO_OLD_POLY1305
+
+For compiling out special characters that embedded devices may have problems with.
+
+#### NO_HANDSHAKE_DONE_CB
+
+Applys a restriction on secure negotation. Don't allocate Suites' object on renegotiation.
+
+#### NO_STDIO_FILESYSTEM
+
+ASN Library Selection (default to ASN_TEMPLATE).
+
+#### NO_TLS_DH
+
+Excludes TLS DH. Should not negotiate cipher suites based on ephemeral finite-field Diffie-Hellman key agreement.
+
+#### NO_WOLFSSL_CM_VERIFY
+
+Sets certification manager callback.
+
+#### NO_WOLFSSL_DIR
+
+Disable directory support.
+
+#### NO_WOLFSSL_RENESAS_TSIP_TLS_SESSION
+
+For disabling only the TSIP TLS-linked Common key encryption method.
+
+#### NO_WOLFSSL_SHA256
+
+Disables support for SHA-256.
```

Comment:
This applies to TLS v1.3 only. It allows SHA2-256 to be enabled and usable from wolfCrypt, but exclude it from TLS v1.3.